apps/web/components/header.tsx

```diff
@@ -14,7 +14,11 @@ export const Header = () => {
           alt="BolhaDEV.chat logo featuring a stylized globe and a speech bubble with a language symbol on a dark background."
         ></Image>
       </Link>
-      <Link href="https://github.com/brunocroh/bolhadev.chat">
+      <Link
+        href="https://github.com/brunocroh/bolhadev.chat"
+        target="_blank"
+        className="absolute right-0 top-0"
+      >
         <GithubCorner />
       </Link>
```

Comment:
Now this component is outside the bounds of the header, so move it outside the header, the main layout component will be a good place